fluidBuild.config.cjs

```diff
@@ -157,6 +157,71 @@ module.exports = {
 			script: false,
 		},
 	},
+
+	multiCommandExecutables: ["oclif", "syncpack"],
+	declarativeTasks: {
+		"jssm-viz": {
+			inputGlobs: ["src/**/*.fsl"],
+			outputGlobs: ["src/**/*.fsl.svg"],
+		},
+		"oclif manifest": {
+			inputGlobs: ["package.json", "src/**"],
+			outputGlobs: ["oclif.manifest.json"],
+		},
+		"oclif readme": {
+			inputGlobs: ["package.json", "src/**"],
+			outputGlobs: ["README.md", "docs/**"],
+		},
+		"syncpack lint-semver-ranges": {
+			inputGlobs: [
+				"syncpack.config.cjs",
+				"package.json",
+
+				// release group packages; while ** is supported, it is very slow, so these entries capture all the levels we
+				// have packages at today. Once we can upgrade to a later version of
+				// globby things might be faster.
+				"{azure,examples,experimental,packages}/*/*/package.json",
```

Comment:
build-tools doesn't need to be here?

Reply:
These paths are interpreted relative to the release group root, so the "packages" entry covers all the other release groups; they all put all their packages under "packages."

I'll double-check though, because I might also be thinking about my refactored code from another PR.

Reply:
To be more precise, the globs are interpreted relative to the package the task belongs to, which in the case of root
tasks like syncpack, is the release group root.